docs/cloud-workload-security/secl.json

```diff
@@ -2,7 +2,7 @@
   "event_types": [
     {
       "name": "*",
-      "definition": "",
+      "definition": "SECLDoc[event.async] Definition:`True if the syscall was asynchronous`",
```

Comment:
I think this indicates an issue if this changes no ?

Reply:
Yes, I'm going to have a look

Reply:
It should be ok now

Reply:
thanks !